test/unit/object/test_item.py

```diff
@@ -141,3 +141,43 @@ def test_get(test_item_and_response, mock_box_session, fields, mock_object_id, e
     mock_box_session.get.assert_called_once_with(expected_url, params=expected_params, headers=if_none_match_header)
     assert isinstance(info, test_item.__class__)
     assert info.id == mock_object_id
+
+
+def test_add_to_collection(test_item_and_response, mock_box_session, mock_collection):
+    # pylint:disable=redefined-outer-name, protected-access
+    test_item, mock_item_response = test_item_and_response
+    expected_url = test_item.get_url()
+    expected_params = {'fields': 'collections'}
+    expected_data = {
+        'collections': [{'id': mock_collection.object_id}]
+    }
+    mock_response = {
+        'collections': []
+    }
+    mock_box_session.get.return_value.json.return_value = mock_response
+    mock_box_session.put.return_value = mock_item_response
+
+    test_item.add_to_collection(mock_collection)
+
+    mock_box_session.get.assert_called_once_with(expected_url, params=expected_params)
+    mock_box_session.put.assert_called_once_with(expected_url, data=json.dumps(expected_data))
+
+
+def test_remove_from_collection(test_item_and_response, mock_box_session, mock_collection):
+    # pylint:disable=redefined-outer-name, protected-access
+    test_item, mock_item_response = test_item_and_response
+    expected_url = test_item.get_url()
+    expected_params = {'fields': 'collections'}
+    expected_data = {
+        'collections': []
+    }
+    mock_response = {
+        'collections': [{'id': mock_collection.object_id}]
```

Comment:
Can you add tests where:
- The collections list is empty.
- The collections list is non-empty, but doesn't contain the given collection.
- The collections list contains the given collection and another collection.